mw-webapp/src/component/button/Button.cy.tsx

```diff
@@ -0,0 +1,36 @@
+import {Button} from "src/component/button/Button";
+
+const BUTTON_VALUE = "Click button";
+
+describe("<Button />", () => {
```

Comment:
Let's skip test which check possibility to render

Reply:
Deleted